src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -51,8 +51,10 @@ end
     return index_end, 0
   elseif index === :i_forward
     return index_begin, 1
-  else # if index === :i_backward
+  elseif index === :i_backward
     return index_end, -1
+  else
+    error("Invalid identifier $index: Only :begin, :end, :i_forward, :i_backward are valid index identifiers")
```

Comment:
Does this check have performance implications? Maybe it's not a performance-critical method. But I have seen performance regressions in the past that were caused by one of the branches being a no-return branch, thus I am just curious.

Reply:
We can't find a difference when benchmarking only this function. But we accidentally used invalid identifiers multiple times, and it is really hard to debug this if the simulation is running and just returning too big errors.

Reply:
Sounds good, then keep it as it is!